bib/cmd/bootc-image-builder/image.go

```diff
@@ -22,12 +22,11 @@ type ManifestConfig struct {
 	Config       *BuildConfig
 	Repos        []rpmmd.RepoConfig
 	Architecture arch.Arch
-	Seed         int64
 }
 
 func Manifest(c *ManifestConfig) (*manifest.Manifest, error) {
 
-	source := rand.NewSource(c.Seed)
+	source := rand.NewSource(int64(0))
```

Comment:
Let's actually make it random now?  If people are using the tool already, let's not have them generating images with identical UUIDs.

Reply:
Done. ;)